documentation/docs/steps/handlePipelineStepErrors.md

```diff
@@ -25,13 +25,13 @@ handlePipelineStepErrors (stepName: 'executeHealthCheck', stepParameters: parame
 
```

Comment:
I would suggest to remove thins whole section and simple link to the actual template file in the step description. What do you think?

Reply:
Yes, but which template file?

Reply:
Also, my intention is not to modify the documentation, but of course we can do it.

Reply:
Lets fix the build first ;)